jcl/.classpath

```diff
@@ -8,7 +8,7 @@
 	<classpathentry excluding="**/module-info.java" kind="src" path="src/openj9.jvm/share/classes"/>
 	<classpathentry excluding="**/module-info.java" kind="src" path="src/jdk.management/share/classes"/>
 	<classpathentry excluding="**/module-info.java" kind="src" path="src/openj9.cuda/share/classes"/>
-	<classpathentry excluding="**/module-info.java" kind="src" path="src/com.ibm.dataaccess/share/classes"/>
+	<classpathentry excluding="**/module-info.java" kind="src" path="src/openj9.dataaccess/share/classes"/>
```

Comment:
This file on the other hand is missing a copyright.  Can you add the copyright header to it?

Reply:
Added.